cirq/google/api/v2/program.proto

```diff
@@ -198,3 +216,81 @@ message ArgFunction {
   // The arguments to the function.
   repeated Arg args = 2;
 }
+
+// An operation that describes a circuit.
```

Comment:
I would add more description here.  Especially since this doesn't actually describe a circuit.  It references a circuit and then modifies it according to a mapping.

Reply:
Done.